Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'okcomputer'
 gem 'pg' # postgres database
 gem 'propshaft' # The modern asset pipeline for Rails [https://github.com/rails/propshaft]
 gem 'pry' # make it possible to use pry for IRB
-gem 'rails', '~> 7.0.8' # TODO: https://github.com/sul-dlss/pre-assembly/issues/1359
+gem 'rails', '~> 7.1'
```

Comment:
Would you consider pinning to 7.1.0 so we can control when Rails upgrades happens? (Rails doesn't follow semver, so a minor bump can involve major work.)